strict equality of floating point number in testFlipLr

```
self = <test_functional.TestFunctionalAPI testMethod=testFlipLR>

    def testFlipLR(self):
        original_x = self.random_xytp[0][0, 0].copy()
    
        events, images = F.flip_lr_numpy(
            self.random_xytp[0],
            images=self.random_xytp[1],
            sensor_size=self.random_xytp[2],
            ordering=self.random_xytp[3],
            multi_image=self.random_xytp[4],
            flip_probability=1.0,
        )
    
        print(self.random_xytp[2], self.random_xytp[0][0, 0])
        print(events[0, 0])
    
        self.assertEqual(
            self.random_xytp[2][0] - self.random_xytp[0][0, 0],
            original_x,
>           "When flipping left and right x must map to the opposite pixel, i.e. x' = sensor width - x",
        )
E       AssertionError: 31.38062971396687 != 31.380629713966858 : When flipping left and right x must map to the opposite pixel, i.e. x' = sensor width - x

test/numpy_test/test_functional.py:30: AssertionError
```

--- a/README.md
+++ b/README.md
@@ -44,3 +44,17 @@ pre-commit install
 ```
 
 This will install the [black formatter](https://black.readthedocs.io/en/stable/) to a pre-commit hook. When you use ```git add``` you add files to the current commit, then when you run ```git commit``` the black formatter will run BEFORE the commit itself. If it fails the check, the black formatter will format the file and then present it to you to add it into your commit. Simply run ```git add``` on those files again and do the remainder of the commit as normal.
+
+## Run tests
+
+To install pytest
+
+```
+pip install pytest
+```
+
+To run the tests, from the root directory of the repo
+
+```
+python -m pytest test/
+```

--- a/test/numpy_test/test_functional.py
+++ b/test/numpy_test/test_functional.py
@@ -1,4 +1,5 @@
 import unittest
+import numpy as np
 
 import spike_data_augmentation.functional as F
 
@@ -24,8 +25,11 @@ class TestFunctionalAPI(unittest.TestCase):
         print(self.random_xytp[2], self.random_xytp[0][0, 0])
         print(events[0, 0])
 
-        self.assertEqual(
-            self.random_xytp[2][0] - self.random_xytp[0][0, 0],
-            original_x,
+        same_pixel = np.isclose(
+            self.random_xytp[2][0] - self.random_xytp[0][0, 0], original_x
+        )
+
+        self.assertTrue(
+            same_pixel,
             "When flipping left and right x must map to the opposite pixel, i.e. x' = sensor width - x",
         )

